src/coreclr/tools/Common/TypeSystem/IL/Stubs/PInvokeILEmitter.cs

```diff
@@ -380,6 +388,19 @@ private PInvokeILStubMethodIL EmitIL()
             if (_targetMethod.HasCustomAttribute("System.Runtime.InteropServices", "LCIDConversionAttribute"))
                 throw new NotSupportedException();
 
+            if (_callingConvention == UnmanagedCallingConventions.Swift)
```

Comment:
Is it possible to `#ifdef` this branch on platforms where swift interop is not supported?

Reply:
We could condition this out based on target details (the managed code is shared for all targets, so not an ifdef).